src/systems/geometry.js

```diff
@@ -132,6 +132,7 @@ function toBufferGeometry (geometry, doBuffer) {
   if (!doBuffer) { return geometry; }
 
   bufferGeometry = new THREE.BufferGeometry().fromGeometry(geometry);
+  bufferGeometry.oldMetadata = {type: geometry.type, parameters: geometry.parameters || {}};
```

Comment:
No preference – only precedent I could find for this was `mesh.userData[*]` but it would be a little more complicated to pass this metadata up to the Mesh probably